daemons/compute/server/servers/server_default.go

```diff
@@ -465,13 +652,39 @@ func (s *defaultServer) Delete(ctx context.Context, req *pb.DataMovementDeleteRe
 	}, nil
 }
 
-func (s *defaultServer) findRabbitRelativeSource(ctx context.Context, req *pb.DataMovementCreateRequest) (string, error) {
+func (s *defaultServer) findDestinationLustreFilesystem(ctx context.Context, dest string) (*lusv1alpha1.LustreFileSystem, error) {
 
-	computeMountInfo, err := s.findComputeMountInfo(ctx, req)
-	if err != nil {
-		return "", err
+	if !filepath.IsAbs(dest) {
+		return nil, fmt.Errorf("Destination must be an absolute path")
+	}
+	origDest := dest
+	if strings.TrimRight(dest, "/") == dest {
+		dest += "/"
+	}
+
+	lustrefsList := &lusv1alpha1.LustreFileSystemList{}
+	if err := s.client.List(ctx, lustrefsList); err != nil {
+		return nil, fmt.Errorf("Unable to list LustreFileSystem resources")
```

Comment:
Should we include the err error in the returned error? Something like `fmt.Errorf("Unable to list LustreFileSystem resource. Internal Error: %s", err.String())`

Not sure it's entirely useful, but it might come in handy